test/src/example.h

```diff
@@ -205,4 +205,22 @@ class ExampleAbstract : public Object {
 	static void _bind_methods() {}
 };
 
+class ExamplePureVirtualBase : public Object {
+	GDCLASS(ExamplePureVirtualBase, Object);
```

Comment:
```suggestion
class ExampleAbstractBase : public Object {
	GDCLASS(ExampleAbstractBase, Object);
```
More descriptive and correct terminology